clients/java/src/main/java/io/camunda/zeebe/client/impl/response/DecisionImpl.java

```diff
@@ -41,6 +42,17 @@ public DecisionImpl(final DecisionMetadata metadata) {
         metadata.getTenantId());
   }
 
+  public DecisionImpl(final DecisionDefinitionItem item) {
```

Comment:
❓ I have seen some classes as DecisionDefinition (Sort,Filter,Query etc..). 
This one is called `DecisionImpl`, to keep the name consistency, would be better call this one `DecisionDefinitionImpl`?

Reply:
This is an existing class in Zeebe client, here I just added a constructor to this class

Reply:
Thank you @houssain-barouni 
I see, you mean the the ones located to general response, and not `api>search>response`
I have a similar approach on Search Requirement, now it made me raise a question, also feel free to think

💭  This gonna make us to have responses in 2 different directories, and not all responses related to the search would be available on this package: 
<img width="353" alt="image" src="https://github.com/user-attachments/assets/f021b27f-4a16-450d-a3b5-a41a046b0a18">

I have no strong feeling over here, and to be honest I dont know if the best approach is have the DecisionDefinition also defined over here, in order to make it clear to navigate, but also I understand the overhead of "duplicated" code.

What do you think?

Reply:
yes it makes sense, the search response can have more fields than the base response.
I will check how I can sort it out

Reply:
done in https://github.com/camunda/camunda/pull/20774/commits/b776e111033a3cbba6b819ce41023ae9fe7ec1be